src/analytics.ts

```diff
@@ -5,17 +5,16 @@ import { TCoreAttributes, TEvents } from './types'
 type Options = {
     growthbookKey?: string
     growthbookDecryptionKey?: string
-    rudderstackKey: string
-    enableDevMode: boolean
+    rudderstackKey?: string
 }
 
 export function createAnalyticsInstance(options?: Options) {
     let _growthbook: Growthbook, _rudderstack: RudderStack
 
-    const initialise = ({ growthbookKey, growthbookDecryptionKey, rudderstackKey, enableDevMode }: Options) => {
+    const initialise = ({ growthbookKey, growthbookDecryptionKey, rudderstackKey }: Options) => {
         _rudderstack = RudderStack.getRudderStackInstance(rudderstackKey)
```

Comment:
if `rudderstackKey` is optional, then someone initialising without rudderstack key breaks the code here, either we should keep rudderstackKey required or if not, then should at least put the condition here to only instantiate rudderstack if we have the key available

The reason is, one could be initialising the growthbook and don't want rudderstack to be initialised, in this case the package will throw the error like `Error: initialising rudderstack with incorrect write key`

Reply:
that's correct. after conversation I have to update it